go.graphics.swing/src/main/java/go/graphics/swing/sound/SwingSoundPlayer.java

```diff
@@ -28,9 +23,14 @@
 import javax.sound.sampled.LineUnavailableException;
 import javax.sound.sampled.SourceDataLine;
 
+import go.graphics.sound.ForgettingQueue;
+import go.graphics.sound.ForgettingQueue.Sound;
+import go.graphics.sound.ISoundDataRetriever;
+import go.graphics.sound.SoundPlayer;
+
 public class SwingSoundPlayer implements SoundPlayer {
 	private static final int BUFFER_SIZE = 4048 * 4;
-	private static final int SOUND_THREADS = 3;
+	private static final int SOUND_THREADS = 30;
```

Comment:
I thought this thread number would be only for PCs and not for phones, because it is within "swing" and not within "android"